aspnetcore/security/index.md

```diff
@@ -43,6 +43,31 @@ ASP.NET Core and EF contain features that help you secure your apps and prevent
 
 There are more vulnerabilities that you should be aware of. For more information, see the other articles in the **Security and Identity** section of the table of contents.
 
+## Secure authentication flows
+
+We recommend using the most secure secure authentication option. For Microsoft services, the most secure authentication is [managed identities](/entra/identity/managed-identities-azure-resources/overview).
+
+Resource Owner Password Credentials Grant:
+
+* Exposes the user's password to the client.
+* Is a significant security risk.
+* Should only be used when other authentication flows are not possible.
```

Comment:
A person skim-reading this doc might read:

 * The header saying "Secure authentication flows"
 * The bulleted-list subheader saying "Resource Owner Password Credentials Grant"

... and from that conclude "ROPC is the secure authentication flow".

Am I right in understanding that's the opposite of the message you want to convey? If so maybe the subheading should be rephrased as "Avoid Resource Owner Password Credentials Grant because:" to help with skim-readability.

Reply:
I agree with the Avoid comment